### PR TITLE
testing/postfwd2: Rename to postfwd and upgrade to 2.01

### PR DESCRIPTION
**Pull request by bratkartoffel**
The executable name changed to "postfwd3", although the package is at
version 2. As the "postwd" and "postfwd2" are just symlinks to the new
"postfwd3" command, I decided to rename the package.

This also adds a post-install script to add the required users which I
forgot initially.

### CODE REVIEW

**Comment by Ikke** [comment]
Could you split the move and the upgrade in two separate commits? This helps git rename detection among other things.

**Comment by bratkartoffel** [comment]
@Ikke Done, but the diff doesn't really look better

File: testing/postfwd/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=postfwd
+pkgver=2.01
+pkgrel=0
+pkgdesc="Combines complex postfix restrictions in a ruleset similar to those of the most firewalls"
+url="https://postfwd.org/"
+arch="noarch"
+license="BSD"
+depends="perl-net-server perl-net-dns perl-io-multiplex"
```
Comment (andypost) [inline testing/postfwd/APKBUILD]: To move better to add 2 lines for backward compatibility
```
provides="postfwd2=$pkgver-r$pkgrel" # for backward compatibility
replaces="postfwd2" # for backward compatibility
```

**Comment by Ikke** [comment]
Merged as 0383b52 and 37447de. Closing PR.

**Pull request closed without merging**